### PR TITLE
Add huaweicloud.com website as MFA capable

### PR DESCRIPTION
**Pull request by fcpimenta**
Add Huawei Cloud website as an MFA capable platform with enhanced security.

### CODE REVIEW

File: entries/h/huaweicloud.com.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "Huawei Cloud": {
+    "domain": "huaweicloud.com",
+    "tfa": [
+      "sms",
+      "email",
+      "custom-software"
```
Comment (phallobst) [inline entries/h/huaweicloud.com.json]:
```suggestion
      "totp"
```

**Review by phallobst — CHANGES_REQUESTED** [review]
432K for the logo is a bit much, I reckon. :smile: How about this one:

**Comment by fcpimenta** [comment]
Hi @phallobst,

Do you need me to make any more changes so you can accept the PR?

Tell me, I'll do it right away. Thanks!!

**Comment by phallobst** [comment]
Please replace the SVG logo with the one I included in my comment. It's only a bit over 1K in size, compared to your version of 432K.

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**